packages/server/test/support/fixtures/projects/e2e/cypress/integration/issue_1669_spec.js

```diff
@@ -0,0 +1,9 @@
+describe('issue-1669 undefined err.stack in beforeEach hook', () => {
+  beforeEach(() => {
+    cy.setCookie('foo', '   bar')
```

Comment:
@Bkucera if you merge in `develop`, I believe this was fixed to err correctly and no longer causes an `err.stack === undefined`, you may need to come up with a custom command

Also, just curious, is there a reason why is this an `e2e` test and not just a normal driver test that asserts on the error using `cy.on('fail', (err) => {...})`?

Reply:
if you catch the error in the 'fail' handler, it has not yet hit the logic that reads off .stack 

I'll merge in develop, and try locally without these changes

Reply:
Alright, it did pass without the src changes, so I changed the test that now fails without these changes